examples/simple_key_value_store.rs

```diff
@@ -280,7 +278,8 @@ struct Client {
 impl Client {
     fn new(_bootstrap_peers: Vec<Endpoint>) -> Result<Client, RoutingError> {
```

Comment:
yes they are, and I have another WIP (not yet opened PR, where Im taking that out of the example)  good spotting !